pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -338,22 +356,26 @@ func createPipelineSpec(log logr.Logger, tool string, commitTime int64, jbsConfi
 			{Name: PipelineResultImageDigest},
 			{Name: PipelineResultPassedVerification},
 			{Name: PipelineResultVerificationResult},
-			{Name: PipelineResultGitArchive},
+			// TODO: ### DeployPreBuildSource and Deploy push to git. Currently the former is used for GitArchive results.
+			//			{Name: PipelineResultGitArchive},
```

Comment:
Previously `PipelineResultGitArchive` was storing the result of Deploy's git push while DeployPreBuildSource git push was 'lost'. _Currently_ `PipelineResultGitArchive` is now storing the result of `DeployPreBuildSource`. I haven't (yet) added another pipeline result for the Deploy pipeline to also store the result of its git archiving.

Reply:
I think this can come later. It is not 100% clear if we will be pushing to git at all, or if that will come from repour/PNC.